src/lib.rs

```diff
@@ -411,36 +511,71 @@ fn is_container_built(container_name: &str) -> bool {
         .arg("--no-trunc")
         .arg("--filter")
         .arg(format!("name={}", container_name));
-    let output = command.output().unwrap_or_else(|_| {
-        panic!(
-            "Could not check if container name={} exists",
-            container_name
-        )
-    });
+    let output = command.output()?;
     debug!(
         "is_container_built name={} output: {:?}",
         container_name, output
     );
     let all_output =
         String::from_utf8(output.stdout).expect("Invalid output from docker container ls.");
```

Comment:
Here, too, could you avoid the `expect` and rather propagate the error?
If you switch to `anyhow` at some point, you'll be able to this message as an annotation.